### DISCUSSION TITLE
Duplicate compounds generated by MS-DIAL

### DISCUSSION THREAD

**Discussion by gabrielerocchetti** [systemsomicslab/MsdialWorkbench#355] [Q&A]
Dear All,

following a metabolomics-based workflow on MS-DIAL (I tried different versions from 4.90 to 5.2) based on UHPLC-ESI(+)-Orbitrap Q-Exactive FOCUS annotation, several duplicated compounds are generated in the final dataset. This phenomenon occurs independently from the .msp database used for the identification. Basically, we carry out the annotation excluding the retention time from the final ID score, therefore we are using only the accurate mass values (MS1 and MSMS fragments) for the identification followed by retention time and accurate mass alignment. Please, what is the best method to manage this duplicate values? Usually we consider those with the highest total ID score; however, if this latter is a feasible approach when dealing with MSMS confirmed compounds, on the other hand it's not easy to manage for the only-MS1 (untargeted) annotated features. Are there some parameters that could be controlled to reduce the number of duplicate features? For example: reducing MS1 tolerance, increasing the mass slice width, changing the smoothing level and others?
Thank you very much.
Gabriele

**Comment by yjhcx** [comment]
hello Gabriele
you can use total score&S/N to filter the true results. in omics ,this phenomenon is normal. I think it is recommended to exclude data with only MS1, since you cannot identify it.
chengxi
